unison-src/transcripts/merge.output.md

```diff
@@ -1024,7 +1024,7 @@ Bob, meanwhile, first deletes the term, then sort of deletes the type and re-add
 ``` ucm
 project/bob> view Foo.Bar
 
-  type Foo.Bar = Hello Nat Nat | Baz Nat
```

Comment:
These are expected because we're reusing an existing unique type GUID, which affects constructor ordering.